burn-book/src/SUMMARY.md

```diff
@@ -12,9 +12,23 @@
   - [Tensor](./building-blocks/tensor.md)
   - [Autodiff](./building-blocks/autodiff.md)
   - [Module](./building-blocks/module.md)
+  - [Config]()
+  - [Learner]()
+  - [Metric]()
+  - [Record]()
+  - [Dataset]()
+  - [Optimizer]()
+- [Installation and Setup]()
 - [Import ONNX Model]()
 - [Advanced]()
   - [Custom Training Loops]()
   - [Custom Metric]()
   - [Custom Kernels]()
     - [WGPU]()
+  - [WebAssembly]()
+  - [No-Std]()
+- [Contributing]()
+  - [Tenets]()
+  - [Design Philosophy]()
+  - [Architecture]()
```

Comment:
I think this should be elsewhere, maybe in another book.

Reply:
OK. Maybe we should add this to CONTRIBUTING.md. I'll remove them.